--- src/test/resources/org/checkstyle/autofix/recipe/emptystatement/ifemptystatement/InputIfEmptyStatement.java
+++ src/test/resources/org/checkstyle/autofix/recipe/emptystatement/ifemptystatement/OutputIfEmptyStatement.java
@@ -8,11 +8,12 @@
 
 package org.checkstyle.autofix.recipe.emptystatement.ifemptystatement;
 
-public class InputIfEmptyStatement {
+public class OutputIfEmptyStatement {
 
     public void test(boolean condition, boolean other) {
         if (condition)
-            ; // violation 'Empty statement.'
+        {
+        }
 
         if (other) {
             int x = 1;
